dashboard/of-cloud-dashboard/handler.js

```diff
@@ -3,6 +3,24 @@
 const fs = require('fs');
 const request = require('request');
 
+const handleLogout = (context) => {
+  const headers = {
+    'Set-Cookie': [
+      'openfaas_cloud_token=',
+      `Expires=${new Date().toUTCString()}`,
+      `Domain=${process.env.cookie_root_domain}`,
+      'Path=/',
+    ].join('; ')
+  };
+
+  const content = fs.readFileSync(`${__dirname}/dist/logout.html`).toString();
```

Comment:
Can we do this async please? Node is single-threaded.

Reply:
Done